packages/openapi-generator/src/options/per-service-config.ts

```diff
@@ -0,0 +1,66 @@
+import { existsSync, promises } from 'fs';
+import { relative } from 'path';
+
+const { readFile } = promises;
+
+/**
+ * Represents the service configuration for all services.
+ */
+export interface PerServiceConfig {
```

Comment:
From the name I thought it is a the config for one service, object below. I would perhaps just use a list of ServiceConfig[]. I guess filename is the name of the service specification file?

Reply:
https://github.com/SAP/cloud-sdk-js/pull/1249#discussion_r622987222